pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1071,26 +1072,18 @@ func (n *NetworkPolicyController) Run(stopCh <-chan struct{}) {
 	defer n.addressGroupQueue.ShutDown()
 	defer n.internalNetworkPolicyQueue.ShutDown()
 
-	klog.Info("Starting NetworkPolicy controller")
-	defer klog.Info("Shutting down NetworkPolicy controller")
+	klog.Infof("Starting %s", controllerName)
+	defer klog.Infof("Shutting down %s", controllerName)
 
-	klog.Info("Waiting for caches to sync for NetworkPolicy controller")
-	if !cache.WaitForCacheSync(stopCh, n.podListerSynced, n.namespaceListerSynced, n.networkPolicyListerSynced) {
-		klog.Error("Unable to sync caches for NetworkPolicy controller")
-		return
-	}
+	cacheSyncs := []cache.InformerSynced{n.podListerSynced, n.namespaceListerSynced, n.networkPolicyListerSynced}
 	// Only wait for cnpListerSynced and anpListerSynced when AntreaPolicy feature gate is enabled.
 	if features.DefaultFeatureGate.Enabled(features.AntreaPolicy) {
-		if !cache.WaitForCacheSync(stopCh, n.cnpListerSynced) {
-			klog.Error("Unable to sync CNP caches for NetworkPolicy controller")
-			return
-		}
-		if !cache.WaitForCacheSync(stopCh, n.anpListerSynced) {
-			klog.Error("Unable to sync ANP caches for NetworkPolicy controller")
-			return
-		}
+		cacheSyncs = append(cacheSyncs, n.cnpListerSynced)
+		cacheSyncs = append(cacheSyncs, n.anpListerSynced)
```

Comment:
maybe `cacheSyncs = append(cacheSyncs, n.cnpListerSynced, n.anpListerSynced)`

Reply:
Done, thanks for pointing it out